src/solver/conv_hip_implicit_gemm_3d_grouped_fwd_xdlops.cpp

```diff
@@ -160,7 +160,9 @@ void PerformanceConfigHipImplicitGemm3DGroupFwdXdlops::Init(const ProblemDescrip
                                                               {});
         if(conv_ptrs[i]->IsSupportedArgument(argument_ptr.get()))
         {
-            valid_kernels.push_back(conv_ptrs[i]->GetTypeString());
+            std::string temp = conv_ptrs[i]->GetTypeString();
+            temp.erase(std::remove(temp.begin(), temp.end(), ' '), temp.end());
+            valid_kernels.push_back(temp);
```

Comment:
Minor. Avoid extra copy.
```suggestion
            valid_kernels.push_back(std::move(temp));
```